packages/experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -91,4 +110,15 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * A callback on whether this item can be removed
    */
   canRemoveItem?: (item: T) => boolean;
+
+  /** Map of callback functions related to row drag and drop functionality. */
+  dragDropEvents?: IDragDropEvents;
+
+  /**
```

Comment:
I see the dragDropHelper and dragDropEvents exist in both ISelectedItemsProps and ISelectedItemsListProps.. is this to achieve single /multi drag and drop?

Reply:
Not exactly. Those are needed by both the list and the individual item. Basically, the item registers to say hey I'm doing a drag drop thing and the list is like excellent and keeps track of the overall state.

Reply:
makes sense, thanks